daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SResult.scala

```diff
@@ -76,34 +76,25 @@ object SResult {
       callback: Option[ContractId] => Boolean,
   ) extends SResult
 
-  final case class SResultNeedLocalKeyVisible(
-      stakeholders: Set[Party],
-      committers: Set[Party],
-      // Callback.
-      // returns the next expression to evaluate.
-      // In case of failure the call back does not throw but set machine.ctrl to an SErrorDamlException
-      callback: SVisibleByKey => Unit,
-  ) extends SResult
-
-  sealed abstract class SVisibleByKey
-  object SVisibleByKey {
+  sealed abstract class SVisibleToStakeholders
+  object SVisibleToStakeholders {
```

Comment:
A bit weird  to have the same name for the base type and the case object.

Reply:
It's not a case object.

I didn't change anything here though, only renamed the existing type.

Reply:
My mistake.